cupy/cuda/memory.pyx

```diff
@@ -617,13 +665,12 @@ cdef class SingleDeviceMemoryPool:
 
     cpdef Py_ssize_t _round_size(self, Py_ssize_t size):
         """Rounds up the memory size to fit memory alignment of cudaMalloc."""
-        unit = self._allocation_unit_size
+        cdef Py_ssize_t unit = self._allocation_unit_size
```

Comment:
Memo: this annotation is needed to let Cython avoid inferring the type of `unit` as Python int.